tensorflow/python/ops/math_ops.py

```diff
@@ -356,6 +356,26 @@ def _mul(x, y, name=None):
 @tf_export("math.subtract", "subtract")
 @dispatch.add_dispatch_support
 def subtract(x, y, name=None):
+  """Returns 'x' - 'y' element wise
+  
+  For example:
+  >>> x = tf.constant([32])
```

Comment:
Empty line between `For example` and the doctest (make sure you do this for _every_ doctest you add in the PR)

Reply:
Not solved yet.